### PR TITLE
Fix filters rendering bug

### LINKED ISSUE #73: Rendering bug with the filters
# Description

This is a weird bug that seems to only occur with certain steps, but in the end the filtering container leaves a big gap between the facets.

https://user-images.githubusercontent.com/22064420/131007891-698ddee7-a1ee-4122-9c76-aeb9652d3cc6.mp4

## Reproduction Steps
1. Go to the main parts collection page
2. Expand `Item Type`, `Condition` and `Part or Kit`
3. Select `Part Only` filter under `Part or Kit`
4. Then select `New` under `Condition`
5. Notice that there is now a lot empty space between `Condition` and the `Color` `

### PR DESCRIPTION
**Pull request by dhmacs**
This PR closes #73 

## QA

1. Visit the Vercel PR preview
2. Visit a collection page and verify that the bug described by #73 is fixed

### CODE REVIEW

**Comment by ardelato** [comment]
QA 👍 

No whitespae


## Steps Taken:
1. I repeated the steps listed to create the bug in https://github.com/iFixit/react-commerce/issues/73
2. I went to the main parts collection page
3. I expanded the` Item Type`, `Condition` and `Part or Kit`
4. I selected `Part Only` filter under `Part or Kit`
5. I then selected `New` under `Condition`
6. I confirmed there was no longer any empty space between `Condition` and the `Color`

File: features/collection/components/sections/FilterableProductsSection/FilterList/index.tsx [hunk 1]
```diff
@@ -137,6 +143,7 @@ export const FilterList = chakra(({ className }: CollectionFiltersProps) => {
                   >
                      <VariableSizeList
                         ref={listRef}
+                        // overscanCount={2}
```
Comment (masonmcelvain) [inline features/collection/components/sections/FilterableProductsSection/FilterList/index.tsx:146]: Stray comment

**Comment by masonmcelvain** [comment]
CR ✌🏻 Nice!

**Comment by sterlinghirsh** [comment]
CR :zap:

**Comment by sterlinghirsh** [comment]
I merged it with the stray comment left in. We can drop it in a later pull if we want.

File: features/collection/components/sections/FilterableProductsSection/FilterList/index.tsx (hunk 1 above)
Comment (dhmacs) [reply features/collection/components/sections/FilterableProductsSection/FilterList/index.tsx:146]: Will drop it with next PR 👍

**Pull request merged**